Messenger/MessengerClient/src/main/java/com/swrobotics/messenger/client/MessengerClient.java

```diff
@@ -123,7 +125,15 @@ private void startConnectThread() {
 
                     connected.set(true);
 
-                    for (String listen : new HashSet<>(listening)) {
+                    // Prioritize listening to EVENT_TYPE so that we receive our own events
+                    // when listening to the rest of them
+                    Set<String> listeningCopy = new HashSet<>(listening);
+                    if (listeningCopy.contains(EVENT_TYPE))
+                        listen(EVENT_TYPE);
+
+                    for (String listen : listeningCopy) {
+                        if (listen.equals(EVENT_TYPE))
+                            continue;
```

Comment:
Pls Fix Indentation Here (Just a coding style thing.

Reply:
I don't see anything wrong with the indentation here, which line(s) are incorrect?